Automattic/wp-calypso
# Social previews | Add Instagram preview

Related to 1203820933396971-as-1204571006046520/f

## Proposed Changes

* This PR adds Instagram preview to social previews package

## Testing Instructions

- See https://github.com/Automattic/jetpack/pull/30929

<img width="950" alt="Screenshot 2023-05-25 at 3 09 45 PM" src="https://github.com/Automattic/wp-calypso/assets/18226415/01029208-564e-480b-bd57-9dd5e274aadb">


## Pre-merge Checklist
